### PR TITLE
REF use the ci support files for rendering

### PR DESCRIPTION
**Pull request by beckermr**
OK @regro/auto-tick! Here is the PR I was chatting about that uses the CI support files and more of conda-build to render the metadata.

### CODE REVIEW

**Comment by viniciusdc** [comment]
I saw that you solved the issue of fetching more data than necessary using the `_fetch_static_repo`

**Comment by beckermr** [comment]
The zipfile module is in the python standard library. I'd read up on it there.

File: conda_forge_tick/mamba_solver.py [hunk 1]
```diff
@@ -296,7 +296,7 @@ def _is_recipe_solvable_on_platform(recipe_dir, cbc_path, platform, arch):
     # here we extract the conda build config in roughly the same way that
     # it would be used in a real build
     config = conda_build.config.get_or_merge_config(
-        None, exclusive_config_file=cbc_path, platform=platform, arch=arch,
+        None, platform=platform, arch=arch, variant_config_files=[cbc_path],
```
Comment (beckermr) [inline conda_forge_tick/mamba_solver.py:299]: @wolfv IDK if you have used this code anywhere, but I found this bug in the order the config files are applied. In a real build, the .ci_support files generated by conda-smithy come last. If you have not used this bit of code anywhere else, then no need to worry about this.

**Comment by beckermr** [comment]
@CJ-Wright this is ready for review

**Comment by CJ-Wright** [comment]
@beckermr Have you gotten a chance to run `make_graph` locally with these changes? How was the performance?

**Comment by beckermr** [comment]
I've run a subset of the nodes but not the whole thing. Happy to run it all but it won't be until later.

**Comment by beckermr** [comment]
It's roughly 25% slower.

**Comment by beckermr** [comment]
Ok. This code is definitely a little slower. Given that the graph is currently wrong in many cases, I think it should be merged in any case. 

I have some local changes that need to be added so please don't merge yet.

**Comment by beckermr** [comment]
This PR is ready for review and merge.

**Pull request merged**